op-mode-definitions/restart-serial.xml.in

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="restart">
+    <children>
+      <node name="serial-console">
+        <properties>
+          <help>Restart serial console service</help>
```

Comment:
I wonder if we need to think about this wording to avoid confusion with the console server from `set service console-server` (for which there is no restart command yet).

Reply:
What about: `Restart serial console (ttyS)`

Reply:
restart system console (or system-console) is too far from what it does, the vty stuff handled by the conf_mode side remains in conf_mode (and doesn't need an op-mode reset). 

restart system serial-console is getting verbose but more specific.

I'm of the opinion that the command is different enough already but the help message could be improved, which I'll sort out.